test/lib/ansible_test/_internal/coverage/__init__.py

```diff
@@ -62,15 +81,204 @@ def run_coverage(args, output_file, command, cmd):  # type: (CoverageConfig, str
     intercept_command(args, target_name='coverage', env=env, cmd=cmd, disable_coverage=True)
 
 
+def get_python_coverage_files():  # type: () -> t.List[str]
+    """Return the list of Python coverage file paths."""
+    coverage_dir = ResultType.COVERAGE.path
+    coverage_files = [os.path.join(coverage_dir, f) for f in os.listdir(coverage_dir)
```

Comment:
Feel free to ignore me here but `get_*_coverage_files()` is common between python and powershell, you could just have the 1 function and have

```
def _get_coverage_files(language):
    coverage_dir = ResultType.COVERAGE.path
    coverage_files = [os.path.join(coverage_dir, f) for f in os.listdir(coverage_dir)
                      if '=coverage.' in f and ('=%s' % language) in f]

    return coverage_files

def get_python_coverage_files():
    return _get_coverage_files('python')

def get_powershell_coverage_files():
    return _get_coverage_files('powershell')
```

Reply:
I agree. I was considering refactoring this originally and decided I'd save it for later, as I'd already spent more time on this than I'd hoped. Assuming no other changes need to be made, I can clean this up on a follow-up PR.